code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -333,12 +333,12 @@ Uranium, Contaminated
 
 /obj/item/projectile/bullet/c4570
 	name = ".45-70 FMJ bullet"
-	damage = 0
-	wound_bonus = 18
+	damage = 38
```

Comment:
```suggestion
	damage = 42
```
Let it afford a couple more points, since .45-70 is reserved for cream of the crop type stuff.